egui_extras/src/image.rs

```diff
@@ -59,8 +73,12 @@ impl RetainedImage {
     /// # Errors
     /// On invalid image
     #[cfg(feature = "svg")]
-    pub fn from_svg_str(debug_name: impl Into<String>, svg_str: &str) -> Result<Self, String> {
-        Self::from_svg_bytes(debug_name, svg_str.as_bytes())
+    pub fn from_svg_str(
+        debug_name: impl Into<String>,
+        svg_str: &str,
+        filter: TextureFilter,
```

Comment:
Instead of passing it in by argument to all constructors I suggest we add a `set_texture_filter` member instead. Most users will just want to use a safe default (linear).

Reply:
Good point, I tried to do a chain-able `with_texture_filter` method, because it will need to be called at the creation of the texture, or more accurately, before the texture is uploaded to the GPU, so I think the chaining is convenient.